src/main.ts

```diff
@@ -200,7 +204,11 @@ export class IdsToolbox {
           : JSON.stringify(error);
 
       this.addFact(FACT_FINAL_EXCEPTION, reportable);
-      actionsCore.setFailed(reportable);
+
+      if (this.executionPhase !== "post") {
+        actionsCore.setFailed(reportable);
+      }
```

Comment:
Why? Were we seeing failures in the post action that you want to suppress?

Reply:
For all the actions we have today, it'd be a serious bummer for any failures in Post to cause the overall workflow to fail. Right now, those steps would be collecting some diagnostics info (nix-installer) or pushing remaining paths to the cache.

Since Post is mostly for cleanup steps, I don't think there are many use cases which makes sense to actually fail the overall workflow in Post.

Reply:
Ack, makes sense. Are we warning in those cases? It's not immediately obvious from this function if we are?

Reply:
mmm! I think we probably aren't! Good catch, I'll log a warning.

Reply:
All set, thanks!